pkg/pool-stable/test/ComposableStablePool.test.ts

```diff
@@ -1539,30 +1643,39 @@ describe('ComposableStablePool', () => {
       });
     });
 
-    describe('getRate', () => {
+    describe('getRate and protocol fees', () => {
       const swapFeePercentage = fp(0.1); // 10 %
       const protocolFeePercentage = fp(0.5); // 50 %
 
-      sharedBeforeEach('deploy pool', async () => {
+      sharedBeforeEach('deploy  pool', async () => {
```

Comment:
nit
```suggestion
      sharedBeforeEach('deploy pool', async () => {
```

Reply:
(this can be addressed later on)

Reply:
I'd say this is out of scope for this PR. I'd want to keep the diff for this compared to the previous pool as small as possible.